kanidmd/lib/src/idm/oauth2.rs

```diff
@@ -1669,7 +1688,7 @@ mod tests {
             // System admins
             (
                 "oauth2_rs_scope_map",
-                Value::new_oauthscopemap(UUID_SYSTEM_ADMINS, btreeset!["read".to_string()])
+                Value::new_oauthscopemap(UUID_SYSTEM_ADMINS, btreeset!["groups".to_string()])
```

Comment:
`setup_oauth2_resource_server` is used 18 times in tests, this change needs to be carefully considered

Reply:
As far as I was able to tell, this "read" scope was only used in one test and there was no real advantage of adding a new scope as opposed to replacing the existing one.

Reply:
Yeah, read was just a "placeholder" for if you had a scope or not in one test.